src/winetricks

```diff
@@ -11919,6 +11919,37 @@ load_richtx32()
 
 #----------------------------------------------------------------
 
+w_metadata sapi dlls \
+    title="MS Speech API" \
+    publisher="Microsoft" \
+    year="2011" \
+    media="download" \
+    conflicts="speechsdk" \
+    file1="../win7sp1/windows6.1-KB976932-X86.exe" \
+    installed_file1="${W_SYSTEM32_DLLS_WIN}/sapi.dll"
+
+load_sapi()
+{
+    for stub in "${W_SYSTEM32_DLLS}/Speech" "${W_SYSTEM64_DLLS}/Speech"; do
```

Comment:
Please add a comment here about why this is needed.

Reply:
@austin987 Done.